### ISSUE TITLE
Focus outline not visible for BaseButton in IE11

### ISSUE THREAD

**Issue by mohitbagra** [OfficeDev/office-ui-fabric-react#3803]
### Bug Report

- __Package version(s)__: 
Latest

- __Browser and OS versions__: 
IE11

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? 
Yes

Requested priority: (Blocking, High, Normal, Low)  
Normal

Products/sites affected: (if applicable)  
VSTS

#### Describe the issue:

Focus outline is not visible on buttons in IE11 (Any button extending BaseButton component)
![image](https://user-images.githubusercontent.com/3766956/35400384-98bc1c8e-01c4-11e8-9c29-2e685aa72da0.png)
The issue is more visible in icon button
![image](https://user-images.githubusercontent.com/3766956/35400514-f99ac23a-01c4-11e8-9c9e-d0a949bd700a.png)

In chrome it looks like this -
![image](https://user-images.githubusercontent.com/3766956/35400461-d9403628-01c4-11e8-8309-77b7f8291de0.png)
![image](https://user-images.githubusercontent.com/3766956/35400471-e38e6492-01c4-11e8-8c6d-92d8a45266c3.png)

It seems like this change caused this regression-
https://github.com/OfficeDev/office-ui-fabric-react/commit/8a8c240b39c3ba12effab2c6c8e7021c3bc9cea9#diff-2ba1f2ae618d0fef86a4c03a08260b38

#### Actual behavior:

Focus outline is not visible in IE11

#### Expected behavior:

Focus outline on buttons should be visible in IE11

### If applicable, please provide a codepen repro:

Can be reproed in localhost site from office fabric react repo. Note that this issue is not visible in https://developer.microsoft.com/en-us/fabric#/components/button because this page adds a rule -
button {
   overflow: visible
}
via a non-fabric css which fixes this issue.

**Comment by mohitbagra** [comment]
The fix should be to replace -
getFocusStyle(theme, -1),
with 
getFocusStyle(theme, 0),

in BaseButton.Styles.ts file
